### DISCUSSION TITLE
Can Navidrome run in a portable way? & if not can be run from a portable WAMP server?

### DISCUSSION THREAD

**Discussion by tomasts248** [navidrome/navidrome#2328] [Q&A] [answered]
If any of those alternatives are doable, which would be the steps?
Oj by portable I mean it can run in windows without permanently modifying the system as well as saving all its settings inside the app folder.

**Comment by deluan** [comment, marked as answer]
Yes, it can be run as a portable app. Just download the Windows executable from the Releases page.
Create a folder in your system, save the executable there. When you run the executable, it will by default create its DB in the current folder, and look for music files under a .\music subdirectory.

If you want to point it to a different music folder, create a simple `navidrome.toml` file alongside the executable with the following contents:

```toml
MusicFolder='C:\Users\JohnDoe\Music' # Set to the folder your music is stored
```

For more options you can add to the config file, check https://www.navidrome.org/docs/usage/configuration-options/

**Reply by tomasts248** [reply]
Thank you so much! This leads me to another question: can I specify more than one source (music folder)?

**Reply by deluan** [reply]
Not at the moment. There is an issue open for it: https://github.com/navidrome/navidrome/issues/192
and it is in the roadmap: https://www.navidrome.org/docs/overview/#road-map

**Reply by tomasts248** [reply]
Thank you :+1:
